pvr.hts/addon.xml.in

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.hts"
-  version="22.6.3"
+  version="22.6.4"
```

Comment:
Please increase the minor version number, not the micro. We increase minor whenever we add new functionality.